code/game/objects/items.dm

```diff
@@ -139,7 +139,7 @@ GLOBAL_DATUM_INIT(fire_overlay, /mutable_appearance, mutable_appearance('icons/e
 	var/list/species_exception = null
 
 	///Who threw the item
-	var/mob/thrownby = null
+	var/mob/thrownby = null //I cannot verbally describe how much I hate this var
```

Comment:
Could possibly be a weakref, not sure what would be cheaper here, mobs don't delete that often.

Reply:
weakref might be easier, I'll clear it up tomorrow/a few days from now

Reply:
yeah, it was simple enough, I call it semi often for code  cleanliness, but I don't think any of these  procs are that hot